karabo_data/tests/test_reader_mockdata.py

```diff
@@ -105,23 +105,39 @@ def test_iterate_trains_require_all(mock_sa3_control_data):
         assert tids != []
 
 
-def test_read_fxe_run(mock_fxe_run):
-    run = RunDirectory(mock_fxe_run)
+def test_read_fxe_raw_run(mock_fxe_raw_run):
+    run = RunDirectory(mock_fxe_raw_run)
     assert len(run.files) == 18  # 16 detector modules + 2 control data files
     assert run.train_ids == list(range(10000, 10480))
     run.info()  # Smoke test
 
+def test_read_spb_proc_run(mock_spb_proc_run):
+    run = RunDirectory(mock_spb_proc_run) #Test for calib data
+    assert len(run.files) == 16 # only 16 detector modules for calib data
+    assert run.train_ids == list(range(10000, 10064)) #64 trains
+    tid, data = next(run.trains())
+    device = 'SPB_DET_AGIPD1M-1/DET/15CH0:xtdf'
+    assert tid == 10000
+    assert 'image.data' in data[device]
+    assert 'f4' == data[device]['image.data'].dtype
```

Comment:
Did you run this? I'd be surprised if comparing a dtype object against a string like this works.

Reply:
Yes, I did and it did not complain - I mean the travis checks were also successful, right? But to be honest I was not thinking about it when wrote this, now that you mention it I am surprised that this is working!

Reply:
Oh yes, it did work. I had tried `'u8' == np.uint64`, which doesn't work, but the relevant thing is `'u8' == np.dtype(np.uint64)`, which does.